packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -215,7 +215,20 @@ export class UIFormComponent extends React.Component {
 		const { mergedSchema } = this.state;
 		const { properties, customValidation } = this.props;
 		const newErrors = validateAll(mergedSchema, properties, customValidation);
-		const errors = { ...this.props.errors, ...newErrors };
+		Object.keys(this.props.errors).forEach(errorKey => {
+			// null is defined to identified field without error
+			// undifined correspond to non existing or hidden fields.
+			if (newErrors[errorKey] !== undefined) {
+				newErrors[errorKey] = this.props.errors[errorKey];
+			}
+		});
+		const errors = {};
+		Object.keys(newErrors).forEach(errorKey => {
+			if (newErrors[errorKey]) {
+				errors[errorKey] = newErrors[errorKey];
+			}
+		});
```

Comment:
```javascript
const errors = Object.entries(newErrors)
    .filter(entry => entry[1])
    .reduce((accu, [key, value]) => { accu[key] = value; }, {});
```

Reply:
with that i have 2 lint errors:

Expected to return a value in arrow function
Assignment to property of function parameter 'accu'
it's ok for you to ignore them?